src/app/pages/send/send-crypto-asset-form/components/send-max-button.tsx

```diff
@@ -30,13 +30,8 @@ export function SendMaxButton({ balance, sendMaxBalance, ...props }: SendMaxButt
   if (sendMaxBalance === '0') return <Box height="32px" />;
 
   return (
-    <LeatherButton
-      data-testid={SendCryptoAssetSelectors.SendMaxBtn}
-      onClick={onSendMax}
-      variant="link"
-      {...props}
-    >
+    <Link data-testid={SendCryptoAssetSelectors.SendMaxBtn} onClick={onSendMax} {...props}>
```

Comment:
Maybe we can rename this component as `SendMaxLink` or even replace it and move the logic up? 

The code above seems weird: 
```
if (sendMaxBalance === '0') return <Box height="32px" />
```

Maybe we can just return `null` in this case without the need for a `Box`

Reply:
I think these might be in place for spacing issues but will check.